org.jacoco.cli/about.html

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html>
+<head>
+<title>About</title>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+</head>
+<body lang="EN-US">
+<h2>About This Content</h2>
+
+<p>
+  @build.date@
+</p>
+
+<h3>License</h3>
+
+<p>
+  All Content in this plug-in is made available by Mountainminds GmbH &amp; Co.
```

Comment:
@marchof this is nor OSGi bundle, nor Eclipse plug-in, nor any other plugin 😉  however we also have such file for `org.jacoco.agent.rt` but not for `org.jacoco.maven-plugin`